docs/eco-fleet-api/getting-started.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Getting Started
+description: Overview of Navixy Eco Fleet API
+---
+
+## Navixy Eco Fleet API
+
+The structure of Eco Fleet API is close to the user API, so
+we highly recommend reading [Backend API: getting started](../backend-api/getting-started.md).
+
+The main differences are _request paths_, _authorization system_ and _request format_.
+
+***
+
+### Base URL
+
+Eco Fleet API resides in `eco_fleet` subsection of API URL. So you can determine URL to API calls like this:
+
+*  `https://api.eu.navixy.com/eco_fleet` for European Navixy ServerMate platform.
+*  `https://api.us.navixy.com/eco_fleet` for American Navixy ServerMate platform.
+
+For example, to make a sensor quality API call in European Navixy ServerMate, you should use the URL: 
+```
+https://api.eu.navixy.com/eco_fleet/v1/trackers/123/sensors/321/quality
+```
+***
+
+### Auth
+
+***
+
+#### Authentication
+
+Authentication is handled by [Backend API](../backend-api/how-to/get-api-key.md).
+
+***
+
+#### Authorization
+
+You should pass the session hash you obtained earlier as the `Authorization` HTTP header with `NVX` auth scheme.
+
+Example:
+```shell
+$ curl -X GET 'https://api.domain.com/eco_fleet/v1/trackers/123/sensors/321/quality' \
```

Comment:
I think we should substitute the real domain in the examples so that it is consistent with the Backend API. See `api_example_url` extra variable.

Reply:
That exact variable includes version which is missing from eco_fleet urls. Should I add another variable?

Reply:
Yep. Just add another one variable in `mkdocs.yml`